### PR TITLE
Simplify configuration of git setting safe.directory

### PR DESCRIPTION
**Pull request by aledbf**
## Description

logs: https://cloudlogging.app.goo.gl/Wr2SmPaFz7EVu4zb6

## How to test
- Workspaces should open and work as expected

## Release Notes

```release-note
NONE
```

## Werft options:

- [X] /werft with-preview

### CODE REVIEW

**Comment by roboquat** [comment]
@aledbf: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/content-service/pkg/initializer/git.go [hunk 1]
```diff
@@ -70,15 +70,11 @@ func (ws *GitInitializer) Run(ctx context.Context, mappings []archive.IDMapping)
 	}
 
 	gitClone := func() error {
-		if err := os.MkdirAll(ws.Location, 0775); err != nil {
+		if err := os.MkdirAll(ws.Location, 0755); err != nil {
```
Comment (sagor999) [inline components/content-service/pkg/initializer/git.go:73]: there was a bug when using 755 permission, that is why it was changed to 775

Comment (aledbf) [reply components/content-service/pkg/initializer/git.go]: Fixed

**Comment by jenting** [comment]
Not sure why, but I can't open the workspace 🤔 
```console
Events:
  Type     Reason     Age                   From               Message
  ----     ------     ----                  ----               -------
  Normal   Scheduled  20m                   default-scheduler  Successfully assigned default/ws-ec91d63d-d37f-416b-9d41-eb3aaafcfa18 to aledbf-git
  Normal   Pulling    20m                   kubelet            Pulling image "reg.aledbf-git.preview.gitpod-dev.com:30393/remote/ec91d63d-d37f-416b-9d41-eb3aaafcfa18"
  Normal   Pulled     19m                   kubelet            Successfully pulled image "reg.aledbf-git.preview.gitpod-dev.com:30393/remote/ec91d63d-d37f-416b-9d41-eb3aaafcfa18" in 56.668016695s
  Normal   Created    19m                   kubelet            Created container workspace
  Normal   Started    19m                   kubelet            Started container workspace
  Warning  Unhealthy  50s (x1139 over 19m)  kubelet            Readiness probe failed: Get "http://10.20.248.120:22999/_supervisor/v1/status/content/wait/true": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
```

**Comment by atduarte** [comment]
@aledbf Release Notes content are not relevant for the end user, we should remove.

**Comment by aledbf** [comment]
> Let me know if this is a critical fix and we need to get it into gen55.

It is.

**Comment by aledbf** [comment]
@sagor999 ready for review

**Review by sagor999 — APPROVED** [review]
LGTM
Tested in preview env, workspaces start and stop, untracked changes preserved and restored.

**Review by mustard-mh — APPROVED** [review]

**Pull request merged**